rholang/src/main/bnfc/rholang_0_2.cf

```diff
@@ -0,0 +1,148 @@
+-- Top level contract declaration
```

Comment:
I presume this file is just a copy of the 0.1 grammar and will eventually be altered to match the 0.2 grammar.  As it stands, there are lots of productions that don't appear in the 0.2 spec, missing operators, etc.

Reply:
This is another file that shouldn't have been added. I've removed it.